bin/console

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env php
```

Comment:
This file can be reverted as well, see `.travis/bin/console`. Thanks!

Reply:
done